doc/storage.md

```diff
@@ -509,7 +509,10 @@ lxc storage create pool1 lvm source=/dev/sdX lvm.vg_name=my-pool
    "zfs.use\_refquota" to "true" for the given dataset or set
    "volume.zfs.use\_refquota" to true on the storage pool. The former option
    will make LXD use refquota only for the given storage volume the latter will
-   make LXD use refquota for all storage volumes in the storage pool.
+   make LXD use refquota for all storage volumes in the storage pool. Also you can
+   set "zfs.reserve\_space" for the dataset or "volume.zfs.reserve\_space" for the
```

Comment:
I'd probably say `on the volume` here instead of `for the dataset`.
And `on the storage pool` for the one afterwards.

Reply:
fixed